app/views/users/index.html.erb

```diff
@@ -10,7 +10,9 @@
   <% @users.each do |user| %>
     <tr>
       <td><%= user.email %></td>
-      <td><%= button_to '+', user_contact_requests_path(user), method: :post, id: user.id %></td>
+      <% if not current_user.sent_contact_request(user) %>
+      <td><%=  button_to '+', user_contact_requests_path(user), method: :post, id: user.id  %></td>
```

Comment:
This line should be indented

Reply:
it is isn't it?

Reply:
The indent must be more because of the if statement

Reply:
Yes, it is currently is indented (see the `Files changed` tab).